images/gcloud/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Includes go and gcloud
-FROM golang:1.9
+FROM golang:1.9.3
```

Comment:
Same here. `golang:1.9` maps to the latest 1.9.x. (ref: https://hub.docker.com/_/golang/)

Reply:
The main reason I did this is because similar things have bit us in the past - if you don't `--pull` when you `docker build`, then you can end up with a stale version of the image. 

Also, making the changes explicit in version control helps for tracking down weird failures.